src/emmc.c

```diff
@@ -771,6 +773,11 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
 
     emmc->partitions = g_list_reverse(emmc->partitions);
 
+    /* Reserve 34 blocks for secondary GPT */
```

Comment:
Remove the `34` to be more fitting with the following code. The explanation will still be useful either way.